modules/mobile/docs/schemas/Appointments.yml

```diff
@@ -33,8 +33,6 @@ properties:
       upcomingAppointmentsCount:
         type: number
         description: The number of BOOKED, non-pending appointments in the next upcomingDaysLimit number of days.
```

Comment:
Removed the examples because they don't actually show up in this case. The actual example data for appointments is broken into files that are already not really correct. I attempted to fix them but it wasn't straight forward so I'll call it out of scope for now.